WooCommerce/Classes/ViewRelated/Orders/Order Creation/NewOrder.swift

```diff
@@ -41,6 +66,7 @@ extension NewOrderHostingController: UIAdaptivePresentationControllerDelegate {
             return true
         }
         UIAlertController.presentDiscardChangesActionSheet(viewController: self, onDiscard: { [weak self] in
+            self?.viewModel.discardOrder()
```

Comment:
I think you may need to merge the base branch because this alert now happens on `presentationControllerDidAttemptToDismiss` not really sure why GH does not raise conflicts, maybe it it is smart enough to put the change where it has to? 🤔

Reply:
Thanks for the heads up! I'll merge the base branch just to be safe.